content/articles/setup-and-run-applications-using-typescript-with-aureliajs/index.md

```diff
@@ -0,0 +1,270 @@
+JavaScript has numerous framework that helps you implement your application out of the pos. This includes React.js Vue, Angular, Next.js, Svelte, Node.js, and Aurelia.js. Each framework enables you to achieve your application objectives. Frameworks such as Aurelia.js, React.js Vue, Angular, Next.js, and Svelte are used to create the frontend web application. In this guide, we will learn how to get started and use Aurelia to build web apps.
+
+Aurelia is a JavaScript client framework for web, mobile, and desktop that leverages simple conventions to empower your UI creativity. Aurelia is the only framework that lets you build applications components using vanilla JavaScript/TypeScript. It uses modern JavaScript and HTML to let you get started quickly and build complex apps.
+
+This guide will help you set up and run Applications using TypeScript while still using the Aurelia.js framework.
+
+### Why use TypeScript
+
+On the other hand, Typescript is a superset of JavaScript. Typescript is gaining a lot of popularity among JavaScript developers. It is one of the fast-developing programming languages for building extensive applications.
```

Comment:
It doesn't talk about the "why"

Talk about the type validation thing which differentiates from JS

Reply:
Add any necessary articles to get started